{
  "messages": [
    {
      "error_id": "",
      "references": [
        {
          "column": 24,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-145557/SPMDIR-MUST/RMA/LocalConcurrency-lockallflushlocal-rput_rget-001.c",
          "line": 46
        },
        {
          "column": 24,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-145557/SPMDIR-MUST/RMA/LocalConcurrency-lockallflushlocal-rput_rget-001.c",
          "line": 48
        }
      ],
      "text": "Found put operation/call at results-20250730-145557/SPMDIR-MUST/RMA/LocalConcurrency-lockallflushlocal-rput_rget-001.c:46:24 which is in conflict with get operation/call at results-20250730-145557/SPMDIR-MUST/RMA/LocalConcurrency-lockallflushlocal-rput_rget-001.c:48:24 before release",
      "type": "Local Data Race"
    }
  ]
}
